packages/edit-navigation/README.md

```diff
@@ -31,7 +31,9 @@ The editing experience is provided as a block editor wrapper around the core fun
 The Navigation Editor has two "modes" for _persistence_ ("saving" navigations) and _rendering_:
 
 1. **Default** - navigations are saved to the _existing_ (post type powered) Menus system and rendered using standard Walker classes.
-2. **Block-based** (opt _in_) - navigations continue to be _saved_ using the existing post type system, but non-link blocks are saved (see technical implementation) and _rendered_ as blocks to provide access to the full power of the Navigation block (with some tradeoffs in terms of backwards compatibility).
+2. **Block-based** (opt _in_) - navigations continue to be _saved_ using the existing post type system, but non-link blocks are
+    - saved (see technical implementation) as blocks.
+    - the navigation is rendered using the Navigation block (as opposed to Walker) to provide access to the full power of the Navigation block (with some tradeoffs in terms of backwards compatibility).
```

Comment:
I think it might be worth referring more to the 

1. **Default** - navigation links are saved to the _existing_ menu item post type system and rendered using standard Walker classes.
2. **Block-based** (opt _in_) - navigation links continue to be _saved_ using the existing menu item post type system, but non-link blocks are
    - saved as block HTML in the menu item post type (see the technical implementation section).
    - the entire navigation is rendered using the Navigation block as opposed to Walker (with some tradeoffs in terms of backwards compatibility). Menu items that are blocks are rendered as blocks. Menu items that are links are rendered as Navigation Link blocks.

Reply:
I think we should disambiguate `links` as a term. So

* Navigation item - the individual items. Can be of any type.
* Navigation Link - refers to the `core/navigation-link` block.

Reply:
Yeah, that sounds fair. I only really use menu, when referring to the something related to the Menu screen as it's called currently in WordPress.